src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.operation.Operation;
+import core.basesyntax.operation.OperationHandler;
+import java.util.Map;
+
+public interface OperationStrategy {
+    Map.Entry<Operation, OperationHandler> getOperationHandler(Operation key);
```

Comment:
The return type of `getOperationHandler` method should be `OperationHandler` instead of `Map.Entry<Operation, OperationHandler>`. The purpose of this method is to retrieve a handler for a given operation, not to return a map entry. This will simplify the usage of this method as clients will only receive what they need, the handler.

Reply:
not fixed